common/remote/sockfile.cpp

```diff
@@ -363,8 +363,86 @@ const char *RFCStrings[] =
+            return "RFSERR_NullFileIOHandle";
+        case RFSERR_InvalidFileIOHandle:
+            return "RFSERR_InvalidFileIOHandle";
+        case RFSERR_TimeoutFileIOHandle:
+            return "RFSERR_TimeoutFileIOHandle";
+        case RFSERR_OpenFailed:
+            return "RFSERR_OpenFailed";
+        case RFSERR_ReadFailed:
+            return "RFSERR_ReadFailed";
+        case RFSERR_WriteFailed:
+            return "RFSERR_WriteFailed";
+        case RFSERR_RenameFailed:
+            return "RFSERR_RenameFailed";
+        case RFSERR_ExistsFailed:
+            return "RFSERR_ExistsFailed";
+        case RFSERR_RemoveFailed:
+            return "RFSERR_RemoveFailed";
+        case RFSERR_CloseFailed:
+            return "RFSERR_CloseFailed";
+        case RFSERR_IsFileFailed:
+            return "RFSERR_IsFileFailed";
+        case RFSERR_IsDirectoryFailed:
+            return "RFSERR_IsDirectoryFailed";
+        case RFSERR_IsReadOnlyFailed:
+            return "RFSERR_IsReadOnlyFailed";
+        case RFSERR_SetReadOnlyFailed:
+            return "RFSERR_SetReadOnlyFailed";
+        case RFSERR_GetTimeFailed:
+            return "RFSERR_GetTimeFailed";
+        case RFSERR_SetTimeFailed:
+            return "RFSERR_SetTimeFailed";
+        case RFSERR_CreateDirFailed:
+            return "RFSERR_CreateDirFailed";
+        case RFSERR_GetDirFailed:
+            return "RFSERR_GetDirFailed";
+        case RFSERR_GetCrcFailed:
+            return "RFSERR_GetCrcFailed";
+        case RFSERR_MoveFailed:
+            return "RFSERR_MoveFailed";
+        case RFSERR_ExtractBlobElementsFailed:
+            return "RFSERR_ExtractBlobElementsFailed";
+        case RFSERR_CopyFailed:
+            return "RFSERR_CopyFailed";
+        case RFSERR_AppendFailed:
+            return "RFSERR_AppendFailed";
+        case RFSERR_AuthenticateFailed:
+            return "RFSERR_AuthenticateFailed";
+        case RFSERR_CopySectionFailed:
+            return "RFSERR_CopySectionFailed";
+        case RFSERR_TreeCopyFailed:
+            return "RFSERR_TreeCopyFailed";
+        case RAERR_InvalidUsernamePassword:
+            return "RAERR_InvalidUsernamePassword";
+        case RFSERR_MasterSeemsToHaveDied:
+            return "RFSERR_MasterSeemsToHaveDied";
+        case RFSERR_TimeoutWaitSlave:
+            return "RFSERR_TimeoutWaitSlave";
+        case RFSERR_TimeoutWaitConnect:
+            return "RFSERR_TimeoutWaitConnect";
+        case RFSERR_TimeoutWaitMaster:
+            return "RFSERR_TimeoutWaitMaster";
+        case RFSERR_NoConnectSlave:
+            return "RFSERR_NoConnectSlave";
+        case RFSERR_NoConnectSlaveXY:
+            return "RFSERR_NoConnectSlaveXY";
+        case RFSERR_VersionMismatch:
+            return "RFSERR_VersionMismatch";
+        case RFSERR_SetThrottleFailed:
+            return "RFSERR_SetThrottleFailed";
+        case RFSERR_MaxQueueRequests:
+            return "RFSERR_MaxQueueRequests";
+    }
+
```

Comment:
Another perhaps less intrusive way to do this here could be if cmd > RFCmax then convert cmd into a string based on its value, such as "(RFSERR %u", cmd)